src/App.js

```diff
@@ -1,9 +1,78 @@
 import "./App.css";
 
 const App = () => {
+  [
+    {
+      name: "aynu",
+      image: "./assets/aynu.png",
+      qr: 88,
+    },
+    {
+      name: "beza",
+      image: "./assets/beza.png",
+      rating: 92,
+    },
+    {
+      name: "desta",
+      image: "./assets/desta.png",
+      rating: 87,
+    },
+    {
+      name: "kafa",
+      image: "./assets/kafa.png",
+      rating: 89,
+    },
+    {
+      name: "lomi",
+      image: "./assets/lomi.png",
+      rating: 88,
+    },
+    {
+      name: "maaza",
+      image: "./assets/maaza.png",
+      rating: 5,
+    },
+    {
+      name: "nehi",
+      image: "./assets/nehi.png",
+      rating: 87,
+    },
+    {
+      name: "saba",
+      image: "./assets/saba.png",
+      rating: 87,
+    },
+    {
+      name: "worke",
+      image: "./assets/worke.png",
+      rating: 88,
+    },
+    {
+      name: "zala",
+      image: "./assets/zala.png",
+      rating: 87,
+    },
+    {
+      name: "zenu",
+      image: "./assets/zenu.png",
+      rating: 82,
+    },
+  ];
   return (
     <div className="App">
       <h1>Coffee guide</h1>
+      <div>
+        <h3>CoffeeMatch</h3>
+        <div>
+          {items.map((item) => (
+            <div>
+              <img src={item.image} alt={item.name} />
+              <h2>{item.name}</h2>
+              <h3>{item.qr}</h3>
+            </div>
+          ))}
+        </div>
+      </div>
```

Comment:
Here you probably want to do something like this:

```suggestion
      <CoffeeMatch matches={matches} />
```